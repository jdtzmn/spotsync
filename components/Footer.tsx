import * as React from 'react'
import css from '../styles/components/footer.scss';

const Footer = () => {
  return (
    <div className={css.footerContainer}>
      <div className={css.flexContainer}>
        {/* SPOTSYNC container */}
        <div className={css.footerSubContainer}>
          <h1 className={css.spotsyncTitle}>
            SPOTSYNC
          </h1>
          <a
            className={css.footerLink}
            href="/"
          >
            Home
          </a>
          <a
            className={css.footerLink}
            href="/app"
          >
            App
          </a>
          <a
            className={css.footerLink}
            href="https://github.com/jdtzmn/spotsync"
          >
            GitHub
          </a>
        </div>
        {/* SUPPORT container */}
        <div className={css.footerSubContainer}>
          <h1 className={css.supportTitle}>
            SUPPORT
          </h1>
          <a
            className={css.footerLink}
            href="/"
          >
            Contact
          </a>
          <a
            className={css.footerLink}
            href="https://github.com/jdtzmn/spotsync/issues"
          >
            Report an issue
          </a>
        </div>
      </div>
      {/* Copyright */}
      <p className={css.copyright}>
        Â© 2018 Jacob Daitzman. All rights reserved.
      </p>
    </div>
  );
};

export default Footer
