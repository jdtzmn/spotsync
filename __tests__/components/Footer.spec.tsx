import * as React from 'react'
import { shallow } from 'enzyme'
import Footer from 'components/Navbar'

describe('Footer', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<Footer />)
  })

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
