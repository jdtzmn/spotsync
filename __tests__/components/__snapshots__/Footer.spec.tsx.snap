// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div>
  <div
    data-cy="navbar"
  >
    <h1>
      Spotsync.
    </h1>
  </div>
</div>
`;
